.github/workflows/build-manywheel-images.yml

```diff
@@ -33,6 +33,10 @@ jobs:
       GPU_ARCH_TYPE: cuda
       GPU_ARCH_VERSION: ${{ matrix.cuda_version }}
     steps:
+      - name: "[FB EMPLOYEES] Enable SSH (Click me for login details)"
+        uses: seemethere/add-github-ssh-key@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Nit: can you please move it into a separate PR (as it has nothing to do with feature in question)

Reply:
Sure I am planning to remove this change, since it seems not to work now